gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -88,16 +83,14 @@ import static retrofit.Endpoints.newFixedEndpoint
 @Import([PluginsAutoConfiguration, DeckPluginConfiguration, PluginWebConfiguration])
 class GateConfig extends RedisHttpSessionConfiguration {
 
-  @Value('${server.session.timeout-in-seconds:3600}')
-  void setSessionTimeout(int maxInactiveIntervalInSeconds) {
-    super.setMaxInactiveIntervalInSeconds(maxInactiveIntervalInSeconds)
-  }
-
-  @Value('${retrofit.logLevel:BASIC}')
-  String retrofitLogLevel
+  private final RetrofitServiceProvider retrofitServiceProvider
```

Comment:
is there an interface we can introduce here where we don't need the compile dependency on kork-retrofit?

Reply:
Yes. Added here.  https://github.com/spinnaker/kork/pull/736